packages/tools/replay-tool/src/helpers.ts

```diff
@@ -26,6 +27,11 @@ import {
 import { ReplayCodeLoader, ReplayUrlResolver } from "./replayLoaderObject";
 import { mixinDataStoreWithAnyChannel } from "./unknownChannel";
 
+export interface ReplayToolContainerEntryPoint {
```

Comment:
Needed an `IProvide...`-like interface to avoid an `as` cast when using the entrypoint in `uploadSummary()` below.